book.bib

```diff
@@ -330,8 +330,7 @@ @Book{McElreath2020
   title =	 {Statistical Rethinking},
   publisher =	 {Chapman and Hall/CRC},
   year =	 2020,
-  note =	 {with implementation by Solomon Kurz:
-                  https://bookdown.org/ajkurz/Statistical_Rethinking_recoded/}
+  note =	 {with implementation by Solomon Kurz: https://bookdown.org/ajkurz/Statistical\_Rethinking\_recoded/}
 }
```

Comment:
Apparently, underscore causes problem so I changes the to "/_"

![image](https://user-images.githubusercontent.com/60338854/152646425-15a97a33-6355-44bd-a984-b09ae7efaa2c.png)